egs/librispeech/asr/simple_v1/decode.py

```diff
@@ -0,0 +1,175 @@
+#!/usr/bin/env python3
+
+# Copyright (c)  2020  Xiaomi Corporation (authors: Daniel Povey, Haowen Qiu)
+# Apache 2.0
+
+import logging
+import os
+import sys
+import warnings
+from concurrent.futures import ProcessPoolExecutor
+from pathlib import Path
+import math
+
+import editdistance
+import numpy as np
+import torch
+import torchaudio
```

Comment:
`torchaudio` is never used in this file and can be removed.

Reply:
I'll wait for @qindazhu to merge his decoding PR and then I'll merge mine to avoid merge conflicts - I can optimize imports again before that.